boards/Kconfig

```diff
@@ -1432,14 +1434,6 @@ config ARCH_BOARD_PIC32MZ_STARTERKIT
 
 		See www.microchip.com for further information.
 
-config ARCH_BOARD_NUCLEO_F429ZI
```

Comment:
why delete

Reply:
Thanks for your feedback. I will rework that.